custom/icds_reports/utils/aggregation_helpers/distributed/child_health_monthly.py

```diff
@@ -96,7 +96,7 @@ def _state_aggregation_query(self, state_id):
             start_month_string=start_month_string)
         not_registered = "(agg_availing.is_registered = 0 AND agg_availing.registration_date::date < {start_month_string})::boolean".format(
             start_month_string=start_month_string)
-        seeking_services = "NOT COALESCE({not_registered},'f') AND NOT COALESCE({migrated}, 'f')".format(
```

Comment:
sorry this wasnt clear. I meant using `distinct from` in the "not_registered" and "migrated" definitions. that way they can be dropped in here directly, and  will always be either true or false

Reply:
No, there are not always `true` or `false`, case of `NULL` is also possible in case the forms are not submitted, that's why using `DISTINCT`